libs/vre/pages/search/advanced-search/src/lib/data-access/advanced-search-service/advanced-search.service.ts

```
@@ -260,15 +242,8 @@ export class AdvancedSearchService {
       })
       .pipe(
         takeUntil(this.cancelPreviousCountRequest$), // Cancel previous request
-        switchMap((response: CountQueryResponse | ApiResponseError) => {
-          if (response instanceof ApiResponseError) {
-            // eslint-disable-next-line @typescript-eslint/no-throw-literal
-            throw response; // caught by catchError operator
-          }
-          return of(response.numberOfResults);
-        }),
+        map(response => response.numberOfResults),
         catchError(err => {
-          this._handleError(err);
           return of(0); // return 0 on error
```

If we catch the error in order to return an empty array we should propagate the error to our AppErrorHandler -> handleError 

this._appErrorHandler.handleError(err);
return of(0);

---

not the scope of this PR